changelog/security/2023-10-11-libtirpc-update.md

```diff
@@ -0,0 +1 @@
+- libtirpc ([libtirpc-rhbg-2138317](http://git.linux-nfs.org/?p=steved/libtirpc.git;a=commit;h=4a2d85c64110ee9e21a8c4f9dafd6b0ae621506d), [libtirpc-rhbg-2150611](http://git.linux-nfs.org/?p=steved/libtirpc.git;a=commit;h=f7f0abdf267698de3f74a0285405b1b01f40893b), [libtirpc-rhbg-2224666](http://git.linux-nfs.org/?p=steved/libtirpc.git;a=commit;h=1d2e10afb2ffc35cb3623f57a15f712359f18e75))
```

Comment:
Nit: s/http/https/ ?

Reply:
I tried that, but `git.linux-nfs.org` seems to be misconfigured and both firefox and bing complain about security risks an stuff (most likely their cert expired).